### PR TITLE
Add appointment notification

### PR DESCRIPTION
**Pull request by evbambly**
image

### CODE REVIEW

File: packages/coordinator/src/components/Popup/NotificationPopup.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  buttonApproveText: string;
+  open: boolean;
+  titleFirst: string;
+  titleSecond?: string;
+  className?: string;
+  onClose: () => void;
+  PopupButton?: React.ReactNode;
+};
+
+export function NotificationPopup({
```
Comment (yaron1m) [inline packages/coordinator/src/components/Popup/NotificationPopup.tsx:18]: Don't we already have a popup component in coordinator?

File: packages/coordinator/src/screens/addAppointments/AddAppointmentsScreenContainer.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ export type NewSlots = {
 export default function AddAppointmentsScreenContainer() {
   const [slotsArray, setSlotsArray] = useState<NewSlots[]>([]);
   const [isSaving, setIsSaving] = useState(false);
+  const [showPopup, setShowPopup] = useState(false);
```
Comment (yaron1m) [inline packages/coordinator/src/screens/addAppointments/AddAppointmentsScreenContainer.tsx:16]: I think this should be inside the component

File: packages/coordinator/src/components/Popup/NotificationPopup.tsx (hunk 1 above)
Comment (evbambly) [reply packages/coordinator/src/components/Popup/NotificationPopup.tsx:18]: Popup split into three: 

1. DialogBase skeleton component
2. Popup implementation
3. NotificationPopup implementation

File: packages/coordinator/src/screens/addAppointments/AddAppointmentsScreenContainer.tsx (hunk 2 above)
Comment (evbambly) [reply packages/coordinator/src/screens/addAppointments/AddAppointmentsScreenContainer.tsx:16]: Moved

File: packages/coordinator/src/screens/addAppointments/AddAppointmentsScreen.tsx [hunk 3]
```diff
@@ -39,9 +48,19 @@ export default function AddAppointmentsScreen({
...
+            onClick={handleSave}
+            isLoading={isSaving}
+          />
         )}
       </footer>
+      <NotificationPopup
+        open={showPopup}
+        buttonApproveText={"המשך"}
+        onClose={() => setShowPopup(false)}
+        titleFirst="התור התווסף בהצלחה!"
```
Comment (yaron1m) [inline packages/coordinator/src/screens/addAppointments/AddAppointmentsScreen.tsx:62]: התור נוסף בהצלחה

**Review by yaron1m — APPROVED** [review]

**Pull request merged**